src/Microsoft.DocAsCode.Build.Engine/HostServiceCreatorWithIncremental.cs

```diff
@@ -111,5 +116,29 @@ private void PostCreate(HostService hostService, IEnumerable<FileAndType> files)
                 IncrementalContext.ReportModelLoadInfo(hostService, loadedFiles, BuildPhase.PreBuildBuild);
             }
         }
+
+        private bool ShouldProcessorTraceInfoCore(IDocumentProcessor processor)
```

Comment:
>ShouldProcessorTraceInfoCore [](start = 21, length = 28)

question: why the method is checking incremental, but naming with should processor trace info?